org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/nexus/MonitorTest.java

```diff
@@ -123,7 +123,7 @@ public void testMixture() throws Exception {
 		List<String> perScan = Arrays.asList("z", "monitor1");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "monitor1" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)